UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Common/Models/RadioNavigationalWarning/DTO/RadioNavigationalWarnings.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace UKHO.MaritimeSafetyInformation.Common.Models.RadioNavigationalWarning.DTO
+{
+    public class RadioNavigationalWarnings
```

Comment:
I think DTO's can be excluded from code coverage.

Reply:
excluded from code coverage.